integration/createdeletebeforeactive_test.go

```diff
@@ -0,0 +1,83 @@
+// +build integration
+
+package integration_test
+
+import (
+	"fmt"
+
+	awseks "github.com/aws/aws-sdk-go/service/eks"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/weaveworks/eksctl/pkg/ctl/cmdutils"
+	"github.com/weaveworks/eksctl/pkg/testutils/aws"
+	. "github.com/weaveworks/eksctl/pkg/testutils/matchers"
+)
+
+var _ = Describe("(Integration) Create & Delete before Active", func() {
+	const (
+		initNG = "ng-0"
+		testNG = "ng-1"
+	)
+
+	Describe("when creating a cluster with 1 node", func() {
+		var clName string
```

Comment:
you probably want to initalize this with `clName := clusterName` (rhs being the global config option). maybe a better variable name is `deleteClusterName := clusterName`

Reply:
My primary purpose of updating the name is that it makes it much easier to distinguish in output (and in the AWS console) the clusters from this test vs. the clusters from creategetdelete_test.go.  It was time-consuming to navigate in the web console and figure out which clusters were being managed by which test case.

Reply:
i don't mean use the same name but re-using it as a prefix.

so instead of
```
cmdutils.ClusterName("", "") + "-delb4active"
```
use
```
clusterName + "-delb4active"
```
(that needs changes on various lines considering `clusterName` could be empty)

Reply:
changed the name `clName` to `delb4activeName`.